### PR TITLE
update logo to modernized Julia

### PR DESCRIPTION
**Pull request by cormullion**
Screenshot 2019-12-09 at 09 54 04

### CODE REVIEW

File: docs/src/assets/figures/generate-logo.jl [hunk 1]
```diff
@@ -4,23 +4,20 @@ function main()
     Drawing(500, 500, "/tmp/colorslogo.png")
     # transparent background
     origin()
-    juliadarkercolors = [Luxor.darker_green, Luxor.darker_purple, Luxor.darker_red]
-    julialightercolors = [Luxor.lighter_green, Luxor.lighter_purple, Luxor.lighter_red]
+    juliacolors = [Luxor.julia_green, Luxor.julia_purple, Luxor.julia_red]
```
Comment (kimikage) [inline docs/src/assets/figures/generate-logo.jl]: What about sorting by their lightness (i.e. from dark to light).
```suggestion
    juliacolors = [Luxor.julia_purple, Luxor.julia_red, Luxor.julia_green]
```

File: docs/src/assets/figures/generate-logo.jl [hunk 2]
```diff
@@ -4,23 +4,20 @@ function main()
...
     pts = ngon(O + (0, 30), 110, 3, -pi/2, vertices=true)
     # I don't think I can draw it with compositing modes
     diskradius = 140
     # basic circles
     for (n, p) in enumerate(pts)
-        sethue(juliadarkercolors[n])
+        sethue(juliacolors[n])
         circle(pts[mod1(n-1, 3)], diskradius, :fill)
```
Comment (kimikage) [inline docs/src/assets/figures/generate-logo.jl]: Fortunately, the sorting does not require `-1`.
```
    circle(p, diskradius, :fill)
```

File: docs/src/assets/figures/generate-logo.jl [hunk 3]
```diff
@@ -4,23 +4,20 @@ function main()
...
-        circle(pts[mod1(n-1, 3)], diskradius-15, :fill)
     end
     @layer begin
         for (n, p) in enumerate(pts)
             circle(p, diskradius, :clip)
             sethue(["cyan", "magenta", "yellow"][mod1(n, 3)])
-            circle(pts[mod1(n+1, 3)], diskradius, :fill)
+            circle(pts[mod1(n + 1, 3)], diskradius, :fill)
             clipreset()
         end
```
Comment (kimikage) [inline docs/src/assets/figures/generate-logo.jl]: from dark to light
```
        for n = 1:3
            circle(pts[mod1(1 - n, 3)], diskradius, :clip)
            sethue(["magenta", "cyan", "yellow"][n])
            circle(pts[mod1(-n, 3)], diskradius, :fill)
            clipreset()
        end
```

Comment (cormullion) [reply docs/src/assets/figures/generate-logo.jl]: I'm not sure I understand these suggestions. Do you want the colors in different places, or do you just want to get rid of the `enumerate`?

Comment (kimikage) [reply docs/src/assets/figures/generate-logo.jl]: logo2
Sorry for missing the explanation. You can ignore these suggestions.:smile:

**Comment by kimikage** [comment]
I forgot to say thank you for the "favicon.ico". I really like it.:heart:

**Comment by cormullion** [comment]
> I forgot to say thank you for the "favicon.ico". I really like it.❤️

Thanks ... I'm sorry it came with some small additional cost... :) Still, learning experiences are always welcome!

**Comment by timholy** [comment]
Not sure what the status here is? Of the suggestions above, only the one to sort the colors seems likely to make a difference; how about we decide and then merge this?

I will abstain, as I defer to the superior artistic judgement of the other participants in this conversation.

**Comment by cormullion** [comment]
:) I'm happy with it.

**Comment by timholy** [comment]
Thanks, @cormullion!

**Pull request merged**